### PR TITLE
Improve (outputlib) documentation

### PR DESCRIPTION
**Pull request by joroeder**
This PR is for improving the documentation, especially the outputlib.

Uwe made the suggestion to structure the documentation of the outputlib in more or less two parts. I made a first attempt. Any comments, suggestions, reviewing are very welcome.

Please feel free to directly commit your improvements!

Related issues: #500 #550 ...

### CODE REVIEW

**Comment by joroeder** [comment]
you will find a new active branch on readthedocs:

[https://oemof.readthedocs.io/en/features-improve_docu/oemof_outputlib.html](https://oemof.readthedocs.io/en/features-improve_docu/oemof_outputlib.html)

File: doc/oemof_outputlib.rst [hunk 1]
```diff
@@ -11,26 +11,116 @@ as the requirements to plotting functions greatly depend on individial requireme
...
+the data in the dictionary by using these keys, where "node" is the name of
+the object you want to address.
+Processing the results is the prerequisite for the examples in the following
+sections.
+
+.. _oplref2:
+
+General approach
+----------------
+As stated above after processing you will get a dictionary with all result
```
Comment (ckaldemeyer) [inline doc/oemof_outputlib.rst]: I am  not sure but I think there is a comma after "As stated above".

File: doc/oemof_outputlib.rst [hunk 2]
```diff
@@ -11,26 +11,116 @@ as the requirements to plotting functions greatly depend on individial requireme
...
+data. At the first try, the keys of the dictionary might seem to have rather
+complex structure, but very useful as well.
```
Comment (ckaldemeyer) [inline doc/oemof_outputlib.rst]: I am not sure what is meant by this exactly ;-) That the structure is complex but useful?

File: doc/oemof_outputlib.rst [hunk 3]
```diff
@@ -11,26 +11,116 @@ as the requirements to plotting functions greatly depend on individial requireme
...
+
+
+.. _oplref3:
+
+Easy access
+-----------
+
+The outputlib provides some functions which will help you to access your
+results directly via labels, which is helpful especially for small energy
+system.
```
Comment (ckaldemeyer) [inline doc/oemof_outputlib.rst]: energy systems

**Comment by joroeder** [comment]
Thanks a lot for your feedback and review!

File: doc/oemof_outputlib.rst (hunk 3 above)
Comment (joroeder) [reply doc/oemof_outputlib.rst]: Done

File: doc/oemof_outputlib.rst (hunk 2 above)
Comment (joroeder) [reply doc/oemof_outputlib.rst]: yeah, you are right. I removed the sentence.

File: doc/oemof_outputlib.rst (hunk 1 above)
Comment (joroeder) [reply doc/oemof_outputlib.rst]: done

**Pull request merged**